go/kbfs/libkbfs/folder_block_ops.go

```diff
@@ -2422,6 +2457,16 @@ func (fbo *folderBlockOps) clearAllDirtyDirsLocked(
 	}
 	fbo.dirtyDirs = make(map[data.BlockPointer][]data.BlockInfo)
 	fbo.dirtyRootDirEntry = nil
+	fbo.dirtyDirsSyncing = false
```

Comment:
This is all locked so it doesn't matter, but IMO the `dirtyDirsSyncing` and `deferredDirUpdates` assignments should be together. After the for loop below.

Reply:
It has to be done together _before_ the loop, not after, since the deferred functions rely on it being false.  But I'll move the other one up and just copy the value there to a temp var for iterating.

Reply:
So it does, thanks.